examples/connext_dds/dynamic_data_request_reply/cpp/PrimeNumberReplier.cxx

```diff
@@ -0,0 +1,328 @@
+
+#include <math.h>
+#include <iostream>
+#include <vector>
+#include "PrimesType.cxx"
+
```

Comment:
As a general comment, delete the extra 'new lines' that we have in the code. For separations, we should use 1 empty line (eg after the #include section).